apps/artist/queries/server.coffee

```diff
@@ -18,6 +18,9 @@ module.exports = """
         text
         credit
       }
+      _artworks: artworks(size: 50, filter: IS_FOR_SALE) {
+        _id
+      }
```

Comment:
Judging off the context of this PR, I think we can definitely remove this as we don't really use the Criteo product feed stuff that this PR is implementing. That said, not sure if this data is used elsewhere on the artist page, but I doubt it. So 👍  to dropping.